Makefile

```diff
@@ -0,0 +1,32 @@
+KIND_CLUSTER_NAME ?= kind
+VERSION ?= v0.0.4
+IMAGE_REPO?= 
+
+REPO_ROOT = $(shell pwd)
+
+.PHONY: help
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+.PHONY: image
+image:
+	docker buildx build \
+	  --platform linux/amd64 \
+	  --load \
+	  -t $(IMAGE_REPO)/cert-manager-example-approver-policy-plugin:$(VERSION) \
+	  .
```

Comment:
I suggest using plain `docker build`. 
I have never understood how to install buildx and I suppose there will be others like me.

Reply:
This is how we've been using buildx:
https://github.com/GarnerCorp/build-actions/blob/075e5985b1deb54fd6cf733ad2bdbb9ac389274e/scripts/docker-build-push.sh#L44-L51

Reply:
Yeah I think in general we also should adapt that approach of setting up a builder as opposed to using whatever is the default configured builder on people's machines, I've had issues with this in the past.
This seems to be an overkill here, so I've removed the use of the buildx plugin.